### DISCUSSION TITLE
How often are the maps updated?

### DISCUSSION THREAD

**Discussion by alanorth** [streetcomplete/StreetComplete#3268] [Q&A] [answered]
I've been very active in my region in the last few weeks and months, both in StreetComplete and on OpenStreetMap via iD and others. The map I see in StreetComplete is often not up to date with OpenStreetMap. How often do the maps in StreetComplete get updated? If I understand correctly, they are from Jawg?

**Comment by matkoniecz** [comment, marked as answer]
There are two steps:

- update of served map data on Jawg side
- cache on StreetComplete side

Second can be forced by clearing app data, or reinstalling. Uploading all answers before that would be useful

**Comment by riQQ** [comment]
I documented some of my findings in #2479. To me, it looks like there is some problem, but I haven't looked into any further.

**Comment by alanorth** [comment]
Hmm OK, looking on Jawg.io I see dozens of changesets (dozens of buildings, roads, other edits) from three days ago that have not been updated. I guess I have to wait for them to get picked up there first. No use fiddling with the cache in StreetComplete until then!
